jack-sys/Cargo.toml

```
@@ -6,7 +6,7 @@ license = "MIT OR Apache-2.0"
 links = "jack"
 name = "jack-sys"
 repository = "https://github.com/RustAudio/rust-jack/tree/main/jack-sys"
-version = "0.3.0"
+version = "0.3.2"
```

```suggestion
version = "0.3.0"
```
No version 0.3.0 has been released.

---

I released 0.3.0 already lol. I'll quit fiddling with versions for now and update it at the end.